packages/components/src/components/dropdown/display-text.tsx

```diff
@@ -1,19 +1,25 @@
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
-import { getDisplayText, listPropType } from './dropdown';
+import classNames from 'classnames';
+import { getDisplayText, TList } from './utility';
 import Text from '../text';
 
-const DisplayText = ({ className, has_symbol, list, is_align_text_left, placeholder, name, value }) => {
-    const symbol_value_class = value && has_symbol ? `symbols--${value.toLowerCase()}` : null;
+type TDisplayText = {
+    className: string;
+    has_symbol?: boolean;
+    is_title: boolean;
+    list: TList;
+    name?: string;
+    placeholder?: string;
+    value: string | number;
+    is_align_text_left?: boolean;
+};
+
+const DisplayText = ({ className, has_symbol, list, is_align_text_left, placeholder, name, value }: TDisplayText) => {
+    const symbol_value_class = value && has_symbol ? `symbols--${value.toString().toLowerCase()}` : null;
     return (
         <React.Fragment>
             {has_symbol ? (
-                <span
-                    className={classNames('symbols dc-dropdown__display-symbol', symbol_value_class, className)}
-                    name={name}
```

Comment:
**name** and **value** are unallowed attributes so I removed them, later I will check this with the QA team!